### PR TITLE
998 update login panel

### LINKED ISSUE #998: New button with PLGRID login
It should be controlled by new env variable, in a similar way as DEMO.
It should handle some kind of redirect, to be later discussed

### PR DESCRIPTION
**Pull request by Derstilon**
Closes #998 , #985 

updated navbar
| image | image | localhost_3000_ (1) |
| - | - | - |
| open | closed | demo |

new conditional button visible only when envirioment variable is set `cross-env REACT_APP_ALT_AUTH=plg npm run start`
image

### CODE REVIEW

**Comment by grzanka** [comment]
@Derstilon does it fix as well this issue https://github.com/yaptide/ui/issues/985 ?

**Review by grzanka — CHANGES_REQUESTED** [review]
adjust selenium tests as well

File: config-overrides.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module.exports = function override(webpackConfig) {
```
Comment (grzanka) [inline config-overrides.js:1]: what is this file doing ?

Comment (Derstilon) [reply config-overrides.js:1]: One new module uses cjs files that aren't supported by create-react-app for some reason.
This fix is based on this issue:
https://github.com/ealmansi/set-interval-async/issues/49

Comment (grzanka) [reply config-overrides.js:1]: Maybe you should add a comment with that explanation ?

File: config-overrides.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+	webpackConfig.module.rules.push({
```
Comment (Derstilon) [inline config-overrides.js:2]:
```suggestion
	// allows create-react-app to load modules with `.cjs` and `.mjs` extension types
	webpackConfig.module.rules.push({
```

**Comment by grzanka** [comment]
Why the left drawer is not aligned to the top ?

Screenshot 2023-05-21 at 14 44 44

**Comment by grzanka** [comment]
Where is logout button ?
I've deployed frontend with master backend. Login works properly, but I don't see a logout button

image

**Review by grzanka — CHANGES_REQUESTED** [review]
Logout button missing

**Comment by grzanka** [comment]
@Derstilon fix conflicts as well

**Comment by Derstilon** [comment]
> Where is logout button ? I've deployed frontend with master backend. Login works properly, but I don't see a logout button
> 
> image

Should be working now

**Review by grzanka — APPROVED** [review]
ok

**Comment by grzanka** [comment]
> > Why the left drawer is not aligned to the top ?
> > Screenshot 2023-05-21 at 14 44 44
> 
> This was stylistic decision. We only have 5 positions on this ui, so I thought it would look better to place it more in the middle. Do you want it back all the way to the top?

It looks good, I was just curious about the reasons

**Comment by grzanka** [comment]
@ostatni5 we are waiting for your review

File: src/WrapperApp/WrapperApp.tsx [hunk 3]
```diff
@@ -23,6 +23,7 @@ import NavDrawer from './components/NavDrawer/NavDrawer';
 import { SimulatorType } from '../types/RequestTypes';
 
 function WrapperApp() {
+	const drawerWidth = 160;
```
Comment (ostatni5) [inline src/WrapperApp/WrapperApp.tsx:26]: Consider using spacing from theme

**Review by ostatni5 — APPROVED** [review]
On demo looks ok

**Review by grzanka — APPROVED** [review]

**Pull request merged**